### ISSUE TITLE
Modifications to run in RHEL 7

### ISSUE THREAD

**Issue by swimboy** [rettenbs/mailman-ldap-memberadaptor#3]
I found that the `try` command to catch the ImportError at line 35 is ineffective when being called from Apache or Postfix. I ended up executing the commands in the `except ImportError` clause unconditionally to get the ldap module to load properly.

Also, for this adaptor to function properly with SELinux enabled, the following boolean needs to be set:

`setsebool -P authlogin_nsswitch_use_ldap 1`

**Comment by rettenbs** [comment]
Thanks for the information.  I am still looking for a nice solution to fix the import. Any idea?

**Comment by swimboy** [comment]
No, I’m not a python guy at all. I had someone else help me figure this out. I’m happy to test for you, though.

        <http://www.tagcomp.com/> 			 	
Jeff McCarty <mailto:jeff@tagcomp.com>, Vice President
jeff@tagcomp.com <mailto:jeff@tagcomp.com>
P.O.Box 1321 • Royal Oak, MI 48068
248.246.7952 office <tel:+1-248-246-7952>
248.736.6866 cell <tel:+1-248-736-6866>
